### PR TITLE
Add Modular code for sql query box (page) and codemirror related pages

### PR DESCRIPTION
**Pull request by Piyush3079**
This pr contains the files needed for modular code for making sql queries.

This pr is in continuation of the pr Mod_Js_Structuring_1. Before reviewing this pr review and merge Mod_Js_Structuring_1

Before submitting pull request, please check that every commit:

- [ ] Has proper Signed-Off-By
- [ ] Has commit message which describes it
- [ ] Is needed on it's own, if you have just minor fixes to previous commits, you can squash them
- [ ] Any new functionality is covered by tests

### CODE REVIEW

**Comment by devenbansod** [comment]
Please rebase

**Comment by Piyush3079** [comment]
This pr is not the next one, next pr is https://github.com/phpmyadmin/phpmyadmin/pull/14524, so we need to first complete this pr(https://github.com/phpmyadmin/phpmyadmin/pull/14524).

**Comment by devenbansod** [comment]
Please rebase.

File: db_search.php [hunk 1]
```diff
@@ -21,9 +21,9 @@
 $response = Response::getInstance();
 $header   = $response->getHeader();
 $scripts  = $header->getScripts();
-$scripts->addFile('db_search.js');
+$scripts->addFile('db_search');
 $scripts->addFile('sql.js');
-$scripts->addFile('makegrid.js');
+// $scripts->addFile('makegrid.js');
```
Comment (devenbansod) [inline db_search.php]: Please remove if not needed

File: js/functions.js [hunk 2]
```diff
@@ -1812,115 +1812,115 @@ function getJSConfirmCommonParam (elem, params) {
...
-        // Copy the sql query to the text area to preserve it.
-        $('#sql_query_edit').text(codemirror_inline_editor.getValue());
-        $(codemirror_inline_editor.getWrapperElement()).off('keydown');
-        codemirror_inline_editor.toTextArea();
-        codemirror_inline_editor = false;
-    }
-    if (codemirror_editor) {
-        $(codemirror_editor.getWrapperElement()).off('keydown');
-    }
+    // $(document).off('click', 'a.inline_edit_sql');
```
Comment (devenbansod) [inline js/functions.js]: Please remove if not needed

File: package.json [hunk 3]
```diff
@@ -24,7 +25,7 @@
     "js-cookie": "2.2.0",
     "sprintf-js": "^1.1.1",
     "tracekit": "0.4.5",
-    "updated-jqplot": "1.0.9",
+    "updated-jqplot": "1.0.9-2",
```
Comment (devenbansod) [inline package.json]: Is such a specific version required?

File: server_sql.php [hunk 4]
```diff
@@ -24,9 +24,9 @@
 $response = Response::getInstance();
 $header   = $response->getHeader();
 $scripts  = $header->getScripts();
-$scripts->addFile('makegrid.js');
-$scripts->addFile('vendor/jquery/jquery.uitablefilter.js');
-$scripts->addFile('sql.js');
+// $scripts->addFile('makegrid.js');
```
Comment (devenbansod) [inline server_sql.php]: Remove if not required

**Review by devenbansod — APPROVED** [review]
LGTM, just a few minor comments.

Great job on this, I acknowledge these files were indeed tricky to manage given their dependencies strewn throughout.

File: package.json (hunk 3 above)
Comment (Piyush3079) [reply package.json]: The previous version was not working with webpack. It was giving some errors related to global scoping. so I have added this updated package.

**Pull request merged**